test/Progress/multi_target_fixture/SConstruct

```diff
@@ -0,0 +1,9 @@
+class ProgressTest:
+
+    def __call__(self, node):
+        if node.get_state() == 2:
```

Comment:
I personally dislike magic numbers - suggest `SCons.Node.executing`

Reply:
yeah was being lazy because test code, but I will make an update.

Reply:
fixed in dd2b96c

Reply:
can probably start using python enums in code now? Since it was added in py3.4?
(of course not suggesting we make that global change in this PR.. ;)
https://docs.python.org/3/library/enum.html